api/apps/v1alpha1/nimcache_types.go

```diff
@@ -71,8 +71,6 @@ type NGCSource struct {
 type ModelSpec struct {
 	// Profiles are the specific model profiles to cache
```

Comment:
We should make this comment more explicit along the lines of "Profiles indicate specific NIM profiles to cache. When these are provided, rest of the model parameters for profile selection are ignored"

Reply:
Fixed